### ISSUE TITLE
Blog link in README is dead

### ISSUE THREAD

**Issue by fusion809** [acrisci/i3-style#41]
Hi,

The blog link ([[1]](http://dubstepdish.com/blog/2013/11/06/introducing-i3-style/)) in your README.md file is dead, I'd like to see the capabilities of this program so please do correct this issue. 

Thanks for your time,
Brenton

**Comment by acrisci** [comment]
Yeah I reset my blog sorry. I didn't save the post, but it's basically what's in the current readme.

**Comment by techtonik** [comment]
http://web.archive.org/web/20161023203717/http://dubstepdish.com/blog/2013/11/06/introducing-i3-style/ and no, there are no screens too (#28).
